rs/ethereum/ledger-suite-orchestrator/src/state/tests.rs

```diff
@@ -153,6 +156,163 @@ mod manage_canister {
     }
 }
 
+mod manage_installed_canisters {
+    use crate::candid::ManageOtherCanisters as CandidManageInstalledCanisters;
+    use crate::scheduler::test_fixtures::{usdc, usdc_metadata, usdt, usdt_metadata};
+    use crate::state::test_fixtures::new_state;
+    use crate::state::{
+        Index, InvalidManageInstalledCanistersError, Ledger, ManageOtherCanisters, State,
+        TokenSymbol,
+    };
+    use assert_matches::assert_matches;
+    use candid::Principal;
+    use maplit::btreeset;
+
+    #[test]
+    fn should_error_when_same_wasm_hash() {
```

Comment:
`should_fail_*` or `should_return_error_*`?

Reply:
ok, renamed the tests in the module.